src/main/java/at/hannibal2/skyhanni/config/storage/ProfileSpecificStorage.java

```diff
@@ -659,4 +659,7 @@ public static class WardrobeStorage {
 
     @Expose
     public UpgradeReminder.CommunityShopUpgrade communityShopProfileUpgrade = null;
+
+    @Expose
+    public int abiphoneContactAmount = -1;
```

Comment:
please init as 0, not -1. we never check against -1 in code, so this adds unnecessary complexity.